all-your-base.md

```diff
@@ -0,0 +1,30 @@
+Implement general base conversion. Given a number on base **a**,
```

Comment:
I have no idea. I'm not a native English speaker.
I will fix that later. Thanks! :smile: